### PR TITLE
Add AdminLTE ul and ol style to the builder.

### PR DESCRIPTION
**Pull request by azraf**
call {!! $AdminNavBar->asUl( array('class' => 'sidebar-menu','style'=>'adminLTE') ) !!} OR {!! $AdminNavBar->asOl( array('class' => 'sidebar-menu','style'=>'adminLTE') ) !!} for slyling admilLTE sidebar dropdown.

### CODE REVIEW

**Comment by leoneljdias** [comment]
Please merge this in master :+1:

File: src/Lavary/Menu/Builder.php [hunk 1]
```diff
@@ -524,7 +532,7 @@ public function render($type = 'ul', $parent = null)
 			}
 					
 			if( $item->hasChildren() ) {
-				$items .= "<{$type}>";
+				$items .= ($this->style == 'adminLTE') ?  "<{$type} class='treeview-menu'>" : "<{$type}>";
```
Comment (dustingraham) [inline src/Lavary/Menu/Builder.php]: Can you generalize this? Such as `$items .= "<".$type.(empty($type_class)?'':" class='{$type_class}'").">"` or something other than a specific treeview-menu class?

File: src/Lavary/Menu/Builder.php [hunk 2]
```diff
@@ -556,6 +568,10 @@ public function asUl($attributes = array())
 	 */
 	public function asOl($attributes = array())
 	{
+                if(!empty($attributes['style']) && (($attributes['style'] == 'adminlte') || $attributes['style'] == 'adminLTE')){
```
Comment (dustingraham) [inline src/Lavary/Menu/Builder.php]: Similar cases here, and at the top. I think adding the ability to include classes on the div is useful, while avoiding the specific AdminLTE implementation.

**Comment by dustingraham** [comment]
Thanks for the PR here. You can now achieve this by using `$children_attributes`

`$MenuName->asUl(['class' => 'parent-menu'],['class' => 'child-menu'])`

In you case something like...

`$MenuName->asUl([],['class' => 'treeview-menu'])`

**Pull request closed without merging**